easybuild/easyconfigs/g/GAMESS-US/GAMESS-US-20220930-R2-gompi-2022a.eb

```diff
@@ -0,0 +1,40 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2017 University of Luxembourg / LCSB, Cyprus Institute / CaSToRC,
+#                                 Ghent University / The Francis Crick Institute
+#
+# Authors::
+# * Kenneth Hoste <kenneth.hoste@ugent.be>
+# * Ward Poelmans <wpoely86@gmail.com>
+# * Oliver Stueker <oliver.stueker@ace-net.ca>
+# License::   MIT/GPL
+##
+name = 'GAMESS-US'
+version = '20220930-R2'
+
+homepage = 'https://www.msg.chem.iastate.edu/gamess/'
+description = """ The General Atomic and Molecular Electronic Structure System (GAMESS)
+ is a general ab initio quantum chemistry package. """
+
+toolchain = {'name': 'gompi', 'version': '2022a'}
+toolchainopts = {'usempi': True, 'openmp': False}
+
+dependencies = [
+    ('OpenBLAS', '0.3.20', '-int8'),
+    ('libxc', '5.2.3'),
+]
+
+# manually download via http://www.msg.chem.iastate.edu/gamess/download.html (requires registration)
+# rename gamess-current.tar.gz by changing 'current' to the proper version
+sources = ['gamess-%(version)s.tar.gz']
+checksums = ['22c8c2dc3ef007c36a32d9ff5a9e1803ac9006f24065455c22de40dd26fb2a7a']
+
+# custom location of gamess workdir
+# scratch_dir = '/path/to/fast/storage'
+# user_scratch_dir = '/path/to/fast/storage'
+
+maxcpus = '256'
+maxnodes = '32'
```

Comment:
Maybe increase those a bit to something that's less likely to be a problem in practice.

Would also be good to reflect these maxima in the module description?

Reply:
sure, fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/19310/commits/cd1391286bc0a231bbf240bf691287fb31fc44fd